plugins/scaffolder-backend/src/lib/templating/filters/parseEntityRef/schema.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { z as zod } from 'zod';
+
+export default (z: typeof zod) => {
```

Comment:
`export const schema`?

I wonder if we should even encourage declaring this inline? Feels weird to split out the schema sometimes in different places as we loose the type inference automatically, and end up having to do awkward stuff like https://github.com/backstage/backstage/blob/1369da8edbf146d4c23da6d05f517193f12b87cc/plugins/scaffolder-backend/src/lib/templating/filters/parseEntityRef/index.ts#L27-L30.

Thinking that we encourage defining the schema and the implementation together, but then the examples can be split out?

Reply:
as you like. I thought I had originally split them at your request 😉